### PR TITLE
add the docs site to the menu

### PR DESCRIPTION
**Pull request by flynnduism**
Screen Shot 2021-01-28 at 9 50 33 AM

Setting this up - adding this link will make the docs.openservicemesh.io site accessible to the public. 
This should not get merged until @phillipgibson gives the 👍 

Signed-off-by: flynnduism <dev@ronan.design>

### CODE REVIEW

**Comment by flynnduism** [comment]
This is good to go once reviewed! @phillipgibson @bridgetkromhout

**Comment by bridgetkromhout** [comment]
@phillipgibson said he is restructuring the docs layout first before we merge this.

**Comment by bridgetkromhout** [comment]
We're waiting for https://github.com/openservicemesh/osm/pull/2471 before merging this.

**Comment by bridgetkromhout** [comment]
Now that https://docs.openservicemesh.io/ is available via https://github.com/openservicemesh/osm/pull/2597, @phillipgibson can let us know here when he's ready for the link to be added to the site.

**Review by phillipgibson — APPROVED** [review]

**Pull request merged**
